src/universal/components/NewMeeting.js

```diff
@@ -288,6 +289,18 @@ class NewMeeting extends Component<Props> {
     })
   }
 
+  onSidebarTransitionEnd = (e) => {
+    const {
+      atmosphere: {eventEmitter},
+      viewer: {
+        team: {isMeetingSidebarCollapsed}
+      }
+    } = this.props
+    if (e.target === this.sidebarRef.current && e.propertyName === 'transform') {
+      eventEmitter.emit('meetingSidebarCollapsed', isMeetingSidebarCollapsed)
```

Comment:
the alternative to this is
- put a cDU on the masonry root
- see if the prop changed, and if it did, reinit the grid after a setTimeout

this is cleaner because if we change the duration of the sidebar toggle, the group grid doesn't break.